api/src/models/schedpredict.model.ts

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+export const scheduleSchema = new Schema(
+  {
+    courses: {type: String, required: true, unique: true},
+    csc_to_seng_ratio: {type: String, required: true, unique: true},
+    class_year_split: {type: String, required: true, unique: true}
+  },
+  {timestamps: true, unique: true}
+);
+
+module.exports = mongoose.model('User', scheduleSchema);
```

Comment:
`mongoose.model('User', scheduleSchema);` Should be new unique name for object. such as `class_size_prediction`. Furthermore im not sure where this data was collected from. I believe maybe google doc. Unfortunately I dont think thats been managed or updated. 
https://github.com/SENG-499-Company-3/backend/blob/dev/api/docs/api_schema.json
Treat this file as your source of truth.